tests/jsrun.py

```diff
@@ -101,7 +102,7 @@ def run_js(filename, engine=None, args=[],
   if not os.path.exists(filename):
     raise Exception('JavaScript file not found: ' + filename)
 
-  command = make_command(filename, engine, args)
```

Comment:
Are any of the changes to this file used in this PR?   Can we revert this file or split out into a separate change?